### PR TITLE
Start workflow with an action

### PR DESCRIPTION
**Pull request by jcoyne**
This allows a deposit to run an action or a notification

### CODE REVIEW

File: app/services/curation_concerns/actors/actor_factory.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ def self.stack_actors(curation_concern)
          ApplyOrderActor,
          InterpretVisibilityActor,
          GrantEditToDepositorActor,
-         InitializeWorkflowActor,
-         model_actor(curation_concern)]
+         model_actor(curation_concern),
+         InitializeWorkflowActor]
```
Comment (straleyb) [inline app/services/curation_concerns/actors/actor_factory.rb]: I see you changed the order of these actors as well. Im curious as to what was going on here with the order of actions and what that might have been causing.

Comment (jcoyne) [inline app/services/curation_concerns/actors/actor_factory.rb]: We need an identifier to be minted before we can initialize workflow.

File: app/services/curation_concerns/workflow/workflow_action_service.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+module CurationConcerns
+  module Workflow
+    class WorkflowActionService
```
Comment (straleyb) [inline app/services/curation_concerns/workflow/workflow_action_service.rb]: Did we want a test for this?

**Comment by straleyb** [comment]
Just had a few questions. Otherwise its good for me. Do we need final approval from others before merging this? @mjgiarlo @jcoyne

**Pull request merged**